src/app/dataset-view/dataset.component.ts

```diff
@@ -210,49 +225,50 @@ export class DatasetComponent
         });
     }
 
-    public onClickNode(idDataset: string): void {
+    public onClickNode(node: any): void {
```

Comment:
We clearly need an interface for those nodes.

Reply:
Yes, i found interface ---> import { Node } from "@swimlane/ngx-graph/lib/models/node.model";